dbfit-java/build.gradle

```diff
@@ -305,23 +310,24 @@ project('informix') {
         'TestDbConnectionDbFitInformixINFORMIX.properties']
 
     dependencies {
-        compile project(':dbfit-java:core')
-        compile 'com.informix:ifxjdbc:4.10.JC4W1@jar'
+        implementation project(':dbfit-java:core')
+        implementation 'org.apache.commons:commons-lang3:3.4'
+        implementation 'com.ibm.informix:jdbc:4.50.4.1@jar'
     }
 }
 
 def publicOracleCompileDeps = [
-    'commons-io:commons-io:2.4'
+    'commons-io:commons-io:2.4@jar'
 ]
 
 project('oracle') {
     description = 'DBFit Oracle engine'
 
     dependencies {
-        compile project(':dbfit-java:core')
-        compile 'dummy:ojdbc6:11.2.0.3.0@jar'
-        compile publicOracleCompileDeps
-        testCompile "org.mockito:mockito-core:${mockitoVersion}"
+        implementation project(':dbfit-java:core')
+		implementation 'dummy:ojdbc8:18.3.0.0.0@jar'		
```

Comment:
ojdbc is a problem as well, we haven't uploaded it to S3 and it's not in official repos. It's better to have separate pull request if we upgrade ojdbc to higher version. ojdbc6.jar for 11.2 is declared to be compatible with JDK8.

Reply:
@javornikolov I've reverted to ojdbc6 and resolved the gradle ivy build problem (with your help). Next problem is the oracle and db2 docker containers don't start correctly. Can I help with this (in master the integration of docker containers for the databases is not present)?

Reply:
@javornikolov DB2 and Oracle docker image is fixed now. I've got a remaining issue with the dbdeploy ant task.

Reply:
@javornikolov dbdeploy and bundle issues are fixed now. Travis CI is ok now.